internal/services/network/ip_group_resource.go

```diff
@@ -47,6 +51,22 @@ func resourceIpGroup() *pluginsdk.Resource {
 
 			"resource_group_name": commonschema.ResourceGroupName(),
 
+			"firewall_ids": {
+				Type:     pluginsdk.TypeSet,
```

Comment:
this should be a type list
```suggestion
				Type:     pluginsdk.TypeList,
```